formbricks/formbricks
# Feature/preview link survey

## What does this PR do?



Adds the option to preview a link survey by adding ?preview=true to the link survey url. It also updates the preview link in the web app accordingly and adds a banner to the top of the previewed link survey.


![image](https://github.com/formbricks/formbricks/assets/42251569/089d7b22-b717-4922-b0f2-f252e95b1333)


## Type of change




## How should this be tested?



- Open a link survey in preview mode, submit it and check if no results appear in the dashboard.
- Open a link survey in preview mode, run a few steps and restart, to see if the restart button is working as expected.
- Open a link survey in production mode, submit it and see if the results appear in the dashboard.

## Checklist
